circle.yml

```diff
@@ -1,11 +1,14 @@
 dependencies:
   pre:
-    - npm install hexo-cli mocha -g
+    - npm install hexo-cli mocha bower gulp -g
     - npm install
+    - bower install
```

Comment:
If you cache maybe bower update is sufficient to avoid downloading everything, but this has to be tested first